content/docs/supportmatrix/_index.md

```diff
@@ -63,6 +63,23 @@ Container Storage Modules doesn't officially support specific operating systems,
 - Ensure the host OS/version aligns with Dell Storage platform support. Check [E-Lab Navigator](https://elabnavigator.dell.com/eln/modernHomeSSM) for details.
 - OS versions not listed must be self-certified using [Cert-CSI](../tooling/cert-csi/). It's recommended to use OS versions that haven't reached end of life.
 
+## CSM Component Versions
+{{<table "table table-striped table-bordered table-sm">}}
+| Component | Version |
+| - | - |
+| CSI PowerFlex | v2.13.0 |
+| CSI PowerStore | v2.13.0 |
+| CSI PowerMax | v2.13.0 |
+| CSI PowerScale | v2.13.0 |
```

Comment:
@gallacher - just wanted to clarify, if this version updates for upcoming v1.14.0 then it should be v2.14.0 version? can you please check and clarify. Thank You.